CHANGELOG.md

```diff
@@ -52,6 +52,13 @@
   
 - Replaced deprecated `stahnma-epel` module with `puppet-epel`. (Enhancement)
   Contributed by @nmaludy
+  
+- Add new parameter `st2::ssl_cert_manage` to allow users to disable this module from
```

Comment:
Looks like this changelog record is part of the `v1.7.0` released version.
I guess the intent was to place it under the `Development` section?

Reply:
Thanks, yeah this was a mistake, good catch!

Reply:
I'm always surprised how even for smallest changes 2 heads are always better than 1.
Glad to be helpful, thanks for requesting a review here!